nixos/modules/virtualisation/sysbox.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.sysbox;
+in
+
+{
+  ###### interface
+
+  options.virtualisation.sysbox = {
+    enable =
+      mkOption {
+        type = types.bool;
+        default = false;
+        description =
+          lib.mdDoc ''
+            This option enables sysbox
+          '';
+        };
+
+    package = mkPackageOption pkgs "sysbox" { };
+  };
+
+  ###### implementation
+
+  config = mkIf cfg.enable {
+    systemd.services.sysbox-mgr = {
+      description = "Sysbox Manager Service";
+      wantedBy = [ "multi-user.target" ];
+
+      path = [ pkgs.rsync pkgs.kmod pkgs.iptables ];
+      script = "${cfg.package}/bin/sysbox-mgr";
```

Comment:
would `lib.getExe` not be more appropriate here? ( and all following uses like this )